common-ui-compose/src/main/java/app/tivi/common/compose/imageloading.kt

```diff
@@ -85,13 +85,13 @@ private val imageSaturationTransitionDef = transitionDefinition {
     }
 
     transition {
-        alpha using tween {
+        alpha using tween<Float> {
```

Comment:
I suspect this is a gradle/AndroidStudio issue.  Flagged as error without this fix, flagged as warning with it :(

Reply:
Yeah, I've been ignoring this since I think it's a Compose bug. Explicitly setting the type is fine, thanks!